kirkeboeger_show3/index.php

```diff
@@ -338,7 +330,13 @@ function lastOne() {
 
 
 
-
+  function changePage(offset) {
```

Comment:
Since this is a single function for all of your buttons, you can easily add more intelligence here if needed (like making sure you don't go off the edge of the book).